src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleCatalogsController.cs

```diff
@@ -39,12 +39,13 @@ ICatalogService catalogService
         /// <remarks>Get common and virtual Catalogs list with minimal information included. Returns array of Catalog</remarks>
 		[HttpGet]
         [Route("")]
-        public async Task<ActionResult<Catalog[]>> GetCatalogs(string sort = null, int skip = 0, int take = 20)
+        public async Task<ActionResult<Catalog[]>> GetCatalogs(string sort = null, int skip = 0, int take = 1000)
         {
             var criteria = AbstractTypeFactory<CatalogSearchCriteria>.TryCreateInstance();
             criteria.Sort = sort;
             criteria.Skip = skip;
             criteria.Take = take;
+            criteria.ResponseGroup = CatalogResponseGroup.Info.ToString();
```

Comment:
@tatarincev 
Should we set minimal response group here according to the method description - "Get common and virtual Catalogs list with minimal information included"(as it is Full by default)?

Reply:
reverted